static/js/biomedical/landing/highlights_section.tsx

```diff
@@ -58,6 +76,40 @@ const Highlight = styled.div`
   }
 `;
 
+const KnowledgeGraphExplorerLink = styled.a`
+  border-left: 1px solid #a0cfcd;
+  color: #1e4e4c;
+  display: flex;
+  flex-direction: column;
+  font-size: 22px;
+  font-weight: 400;
+  gap: 16px;
+  line-height: 28px;
+  padding-left: 24px;
+  text-align: left;
+  width: 340px;
+
+  @media ${BREAKPOINTS.md} {
+    font-size: 16px;
+    line-height: 20px;
+  }
+
+  @media ${BREAKPOINTS.lg} {
+    border-left: none;
+    justify-content: center;
+    text-align: center;
+    width: fit-content;
+  }
+
+  &:hover {
+    text-decoration: none;
+  }
+
+  .icon {
+    font-size: 32px;
+  }
+`;
+
 interface HighlightConfigEntry {
   label: string;
   value: string | number;
```

Comment:
update to just `number`

Reply:
Updated to just number

Reply:
Hi there, 
Regarding this:
https://screenshot.googleplex.com/6Mc3ibgDgLoeqNU
Yes, I agree that is an odd behaviour, that tile should be in line with the rest of the numbers.
If the text wraps it should bit, but all in all a the moment we are missing to fix all the breaking points and margins, so that would help too.